packages/core/template/default/index.md

```diff
@@ -9,124 +9,45 @@
 
 <div class="bg-primary text-white py-5 mb-4">
   <div class="container">
-    <h1 class="display-4 no-index">Landing Page Title</h1>
-    <p class="lead">A tagline can go here</p>
+    <h1 class="display-4 no-index">Great!<br>You've just initialized a MarkBind site.</h1>
+    <p class="lead">Let's get started...</p>
   </div>
 </div>
 
-# Heading 1
-Some text some text some text some text some text some text some text. **Some text some text some text some text some text ==some text== some text**. Some text some text some text some text some text some text some text some text some text some text some text some text some text some text. Some text some text some text some text some text some text. Some text some text some text some text some text some text some text.
+<hr>
 
-**A block quote:**
+##### **User Guide: Authoring Contents**
 
-> Some text some text some text some text some text some text some text. Some text some text some text some text some text some text some text. Some text some text some text some text some text some text some text some text some text some text some text some text some text some text. Some text some text some text some text some text some text. Some text some text some text some text some text some text some text.
+<blockquote>
 
-**A list:**
+Learn about the variety of syntax schemes, formats, and custom MarkBind components that you can use in your MarkBind site.
 
-* item A
-* item B
-  1. item b1
-  1. item b2
-  1. item b3
+</blockquote>
 
-**A `code` example:**
+<span class="indented">More info in: _<a href="https://markbind.org/userGuide/authoringContents.html" target="_blank">User Guide → Authoring Contents</a>_</span>
 
-```html
-<foo>
-  <bar type="name">goo</bar>
-</foo>
-```
+<hr>
 
-## Sub Heading 1.1
+##### **User Guide: Working with Sites**
 
-A <tooltip content=":exclamation: some **important explanation**" placement="top" trigger="hover">tooltip</tooltip>, a <trigger for="modal:modalinfo" trigger="click">modal</trigger>, a <a href="https://markbind.org/">link</a>, a <span class="badge bg-danger">badge</span>, another <span class="badge bg-warning text-dark">badge</span>.
+<blockquote>
```

Comment:
Any problem with `>` instead of `<blockquote>` ? I would prefer Markdown over direct HTML. Same for other content in this page

Reply:
Absolutely no problem! I simply copied what was being done in the [Authoring Contents file](https://github.com/MarkBind/markbind/blob/master/docs/userGuide/authoringContents.md). 

Markdown definitely does make it cleaner - in [44f4454](https://github.com/MarkBind/markbind/pull/2225/commits/44f44542396dd4ab489d92fbe067fcb3498f174a), I've changed the `<blockquote>` to `>`, replaced `<hr>` with `---`, and removed the `<span>`. For now, I kept the blue heading container (to keep the style) and the `<a>` tags (to make use of `target="_blank"`).